lib/logstash/outputs/kafka.rb

```diff
@@ -156,6 +167,16 @@ def create_producer
       props.put(kafka::RETRY_BACKOFF_MS_CONFIG, retry_backoff_ms.to_s)
       props.put(kafka::SEND_BUFFER_CONFIG, send_buffer_bytes.to_s)
       props.put(kafka::VALUE_SERIALIZER_CLASS_CONFIG, value_serializer)
+      
+      if ssl
+        props.put("security.protocol", "SSL")
+        props.put("ssl.truststore.location", ssl_truststore_location)
```

Comment:
I like the exception to bubble up if `ssl` and `ssl_truststore_location` is not set. That's the least required for SSL